Mathlib/Data/Matrix/Basic.lean

```diff
@@ -2253,6 +2253,10 @@ theorem conjTranspose_map [Star α] [Star β] {A : Matrix m n α} (f : α → β
   Matrix.ext fun _ _ => hf _
 #align matrix.conj_transpose_map Matrix.conjTranspose_map
 
+@[simp]
+theorem conjTranspose_eq_transpose [Star α] [TrivialStar α] {A : Matrix m n α} :
```

Comment:
```suggestion
theorem conjTranspose_eq_transpose [Star α] [TrivialStar α] (A : Matrix m n α) :
```
since `A` isn't present in any of the other arguments.